shared/actions/router.js

```diff
@@ -1,42 +1,5 @@
 import * as Constants from '../constants/router'
 
-/*
```

Comment:
I'm not so sure. Here's another side:

The tabs are orthogonal to the router. Components inside a tab shouldn't really care which tab they are in for routing purposes. They should just say navigate to this page, go back, go up.

How often are we navigating to another tab and a different route? and why is it better to combine the notion of switching tabs with the notion of changing routes?